Move jobsets multikueue run details to separte multikueue subfolder (#3135)
kubernetes-sigs/kueue: site/content/en/docs/tasks/run/multikueue/_index.md

---
title: "Using MultiKueue"
linkTitle: "MultiKueue"
weight: 8
date: 2024-09-25
description: >
  The workloads that support MultiKueue.
---

> This page explains how to run tasks in MultiKueue environment.
> 
> For more details about the MultiKueue check the concepts section for a [MultiKueue overview](/docs/concepts/multikueue/). 
> 
> ## Running tasks in MultiKueue environment
> 
> To utilize this feature, you simply submit the job to the Manager cluster, targeting a ClusterQueue that has been configured for MultiKueue operation.
> 
> Once submitted, Kueue automatically handles the delegation of the job to the appropriate worker clusters, requiring no additional configuration changes on your part.

Reviewer: ```suggestion
This page explains how to run tasks in MultiKueue environment for multi-cluster Job dispatching.

In particular, we focus on details of running your tasks using various Job CRDs.
```
TBH I think this is enough here as a starter.
Also currently, "To utilize this feature, you simply submit the job to the Manager cluster, targeting a ClusterQueue that has been configured for MultiKueue operation." is not that simple. details differ. For some CRDs we have managedBy defaulting, for some not, etc.

---

> title: "Run Workloads in MultiKueue"

Reviewer: ```suggestion
title: "Using MultiKueue"
```
shorter, and will read ok both standalone and inside "Run Workloads"

---
title: "Using MultiKueue"
linkTitle: "MultiKueue"
weight: 8
date: 2024-09-25
description: >
  The workloads that support MultiKueue.
---

This page explains how to run tasks in MultiKueue environment.

For more details about the MultiKueue check the concepts section for a [MultiKueue overview](/docs/concepts/multikueue/). 

## Before you begin

> Check the [MultiKueue installation guide](/docs/tasks/manage/setup_multikueue) on how to properly setup MultiKueue clusters.

Reviewer: Please also add that for the integration we recommend using at least Kueue 0.8.1, and JobSet 0.5.2.
Refer to the docs for installing a JobSet.

> ## Running tasks in MultiKueue environment

Reviewer: ```suggestion
## Running JobSet in MultiKueue environment
```

> Any job compatible with Kueue in a single-cluster environment can be seamlessly deployed in a multi-cluster setup using MultiKueue.

Reviewer: Yes, we try to make it as easy as possible, but might not be seamless in practice. For example, if the worker cluster is in another zone or region, then the user needs to solve the storage complications (copy input data, synchronize outputs, etc.).  I would just drop this sentence for now.
